plugins/cpp/service/src/cppservice.cpp

```diff
@@ -307,6 +307,11 @@ void CppServiceHandler::getProperties(
           return_["Abstract type"] = "true";
         if (type.isPOD)
           return_["POD type"] = "true";
+        if (type.size > 0)
+          return_["Size"] = std::to_string(type.size);
```

Comment:
It should be documented somewhere that the unit of this value is in bytes (e.g.: someone writes a new client should know this information) or you can set the unit here not in the client side.
```cpp
return_["Size"] = std::to_string(type.size) + "bytes";
```

Reply:
I didn't want to send it as a string via the api because maybe someone wants to say a different suffix. I'll add this to the docs.